src/js/components/TextInput/TextInput.js

```diff
@@ -270,6 +275,7 @@ if (process.env.NODE_ENV !== 'production') {
 }
 
 export default compose(
+  withFocus,
```

Comment:
THANK YOU! I was hit by this bug this morning.

Reply:
:)